pkg/specgenutil/volumes.go

```diff
@@ -152,7 +153,15 @@ func findMountType(input string) (mountType string, tokens []string, err error)
 	// Split by comma, iterate over the slice and look for
 	// "type=$mountType". Everything else is appended to tokens.
 	found := false
-	for _, s := range strings.Split(input, ",") {
+	csvReader := csv.NewReader(strings.NewReader(input))
+	records, err := csvReader.ReadAll()
+	if err != nil {
+		return "", []string{}, err
+	}
+	if len(records) <= 0 {
+		return "", []string{}, errInvalidSyntax
+	}
```

Comment:
should we error for `len(records) > 1` as well, this will happen if the argument contains a newline?

Reply:
I think we should made this `len(record) != 1`